modules/standard/List.chpl

```diff
@@ -145,7 +147,14 @@ module List {
     /* The type of the elements contained in this list. */
     type eltType;
 
-    /* If `true`, this list will perform parallel safe operations. */
+    /*
+      If `true`, this list will perform parallel safe operations.
+
+      .. warning::
+        ``parSafe`` is unstable. This feature will eventually be replaced by a
+        standalone parallel-safe list type.
+
+    */
```

Comment:
I prefer an unstable pragma here instead of a chpldoc warning.

By analogy, add @unstable on the parSafe field in Map and Set.

Reply:
Please revert to the original format, to avoid an unnecessary diff and for consistency with the comment for `eltType`.